packages/docs/src/components/styleguide.tsx

```diff
@@ -177,21 +141,21 @@ export class Styleguide extends React.Component<IStyleguideProps, IStyleguideSta
         window.addEventListener("hashchange", () => {
```

Comment:
This probably isn't the PR for this, but i think routing should be handled by a separate class